torch/linalg/__init__.py

```diff
@@ -1588,11 +1610,16 @@
 .. seealso::
 
         :func:`torch.linalg.svd` computes the full singular value decomposition.
+        Also check that documentation for the selection of `driver`.
 
 Args:
     A (Tensor): tensor of shape `(*, m, n)` where `*` is zero or more batch dimensions.
 
 Keyword args:
+    driver (str, optional): name of the cuSOLVER method to be used. This keyword argument only works on CUDA inputs.
+        If `None`, a default heuristic of algorithms will be used.
+        Available options are: `None`, `gesvd`, `gesvdj`, `gesvdjBatched`, and `gesvdaBatched`.
```

Comment:
Remove batched options. 
Also, could you also point to the `linalg.svd` function docs here as well?

Reply:
oops, forgot svdvals